resources/src/androidMain/kotlin/dev/icerock/moko/resources/desc/Utils.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.resources.desc
+
+import android.content.Context
+import android.content.res.Resources
+import android.os.Build
+
+object Utils {
+    fun processArgs(args: List<Any>, context: Context): Array<out Any> {
+        return args.toList().map { (it as? StringDesc)?.toString(context) ?: it }.toTypedArray()
```

Comment:
Why `toList()` is called here?

Reply:
good question) it was in old code, just moved in separated class...i will fix this duplication